Description: avoid building a shaded jar with copies of dependencies' classes
Author: Pierre Gruet <pgt@debian.org>
Forwarded not-needed
Last-Update: 2021-05-17

--- a/pom.xml
+++ b/pom.xml
@@ -95,7 +95,7 @@
 					</archive>
 				</configuration>
 			</plugin>
-			<plugin>
+			<!--<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-shade-plugin</artifactId>
 				<version>2.3</version>
@@ -105,9 +105,9 @@
 						<goals>
 							<goal>shade</goal>
 						</goals>
-						<configuration>
+						<configuration>-->
 							<!-- remove all classes of dependencies that are not used by the project -->
-							<minimizeJar>true</minimizeJar>
+							<!--<minimizeJar>true</minimizeJar>
 							<shadedArtifactAttached>false</shadedArtifactAttached>
 							<relocations>
 								<relocation>
@@ -118,7 +118,7 @@
 						</configuration>
 					</execution>
 				</executions>
-			</plugin>
+			</plugin>-->
 			<plugin>
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
